docs/Developers/Review.rst

```diff
@@ -91,6 +91,8 @@ request but should ideally pass for every commit.
 For the Reviewers
 -----------------
 
+-  Read and try to understand the code. If something looks ineffective or
+   bug prone, leave a comment.
```

Comment:
maybe we should also say "even if you doubt that what you say is wrong, just let the code-writer explain what he has thought" or so? not sure...